backend/app.js

```diff
@@ -3,13 +3,24 @@ import cookieParser from "cookie-parser";
 import logger from "morgan";
 import testResource from "./resources/testResource.js";
 
+import authResource from "./resources/authResource.js";
+import otpResource from "./resources/otpResource.js";
+import Connection from "./database/db.js";
+import bodyParser from "body-parser";
+
 const app = express();
 
 app.use(logger("dev"));
-app.use(express.json());
+app.use(express.json())
+app.use(bodyParser.json());;
```

Comment:
consider using either `express.json` or `bodyParser.json` uing both will be redundant.